cppcache/include/geode/PoolFactory.hpp

```diff
@@ -426,6 +426,11 @@ class APACHE_GEODE_EXPORT PoolFactory {
    */
   PoolFactory& addServer(const std::string& host, int port);
 
+  /**
+   * Set proxy info for SNI connection.  Used for connecting via SNI proxy.
+   */
+  PoolFactory& setSniProxy(const std::string& hostname, const int port);
```

Comment:
Deviates from [SNI Proxy RFC](https://cwiki.apache.org/confluence/display/GEODE/Client+side+configuration+for+a+SNI+proxy).

Reply:
""